winch/codegen/src/isa/x64/asm.rs

```diff
@@ -499,4 +484,14 @@ impl Assembler {
             }
         }
     }
+
+    fn load_constant(&mut self, imm: &i64, size: OperandSize) -> Reg {
+        let scratch = regs::scratch();
+        self.mov_ir(*imm as u64, scratch, size);
+        scratch
+    }
+
+    fn handle_invalid_operand_combination(src: Operand, dst: Operand) {
+        panic!("Invalid operand combination; src={:?}, dst={:?}", src, dst);
```

Comment:
I did have to remove the name of the operator. I could add it back as an `&str` parameter if you'd prefer.